File: euler problems/euler1.c
#include <stdio.h>

int main() {
	int adder = 0;
	
	int i;
	
	for (i = 0; i < 1000; i++) {
		if (i % 3 == 0 || i % 5 == 0) {
			printf("%d\t", i);
			adder += i;
		}
	}
	printf("\n%d", adder);
}

File: euler problems/euler1.php
<html>
<head>
<title>php euler</title>
</head>

<body>

<?php

$adder = 0;
$i;
for ($i = 0; $i < 1000; $i++) {
	if ($i % 3 == 0 || $i % 5 == 0) {
		print("\t" . $i);
		$adder += $i;
	}
}

echo "<br> The sum of the multiples of 3 and 5 under 1000 is: " . $adder;

?>

</body>
</html>